tests/test_api.py

```diff
@@ -232,5 +232,18 @@ def test_graphdef(self):
         self.assertAllClose([2.1193342], oy[0], rtol=0.1, atol=0.1)
 
 
+    def test_tflite(self):
```

Comment:
Should we also check the test `@check_tf_min_version("1.15")`, like the other API test?